tests/api/test_agents_mef_rest.py

```diff
@@ -145,6 +145,11 @@ def test_agents_mef_get_updated(client, agent_mef_record, agent_idref_record,
                                 agent_mef_idref_redirect_record,
                                 agent_mef_gnd_redirect_record):
     """Test agents MEF get latest."""
+
+    def utf_isoformat(date):
```

Comment:
Are you sure that no lib can do this?

Reply:
This is the simplest solution for ES DB datetimes.